content/menu.md

```diff
@@ -347,6 +351,9 @@ site_menu:
     url: /develop/api-reference/widgets/st.text_input
     isVersioned: true
   - category: Develop / API reference / Input widgets / MEDIA & FILES
+  - category: Develop / API reference / Input widgets / st.experimental_audio_input
+    url: /develop/api-reference/widgets/st.audio_input
```

Comment:
I can go either way; I think I've waffled on this a little. I definitely want the URL to not have experimental so there are no redirects, and once it's GA, there won't be a separate experimental page since it will just come up automatically in the version selector.

Do you have a preference for that brief time when it's exclusively experimental?

Reply:
I don't have a preference 😅 I trust your judgement

Reply:
I'll put it back since it caught your eye. 😅